java-frontend/src/main/java/org/sonar/java/model/JMethodSymbol.java

```diff
@@ -137,6 +137,10 @@ private void findOverridesInParentTypes(Collection<MethodSymbol> accumulator, Pr
 
   private void findOverridesInTypes(Collection<MethodSymbol> accumulator, Predicate<IMethodBinding> overridesCondition, ITypeBinding... types) {
     for (ITypeBinding type : types) {
+      if (type == null) {
+        // Can happen for unknown reason.
+        continue;
```

Comment:
fix LGTM, but I really would like to find the root cause.
Did you try analyzer jetty on your machine?

Reply:
I managed to reproduce the issue locally when analyzing jetty. The problem comes when analyzing [this line](https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-server/src/main/java/org/eclipse/jetty/server/CustomRequestLog.java#L1095), more specifically when we call `MethodInvocation.symbol().overriddenSymbols()` for the method invocation `getServletContext()`. It goes through the parents just fine until it reaches `AbstractHandler`, where the call to `ITypeBinding.getSuperclass()` returns null for some unknown reasons. It seems to me that digging deeper at this point will take a long time, do you think it is still worth it?